engineapi/types.go

```diff
@@ -106,20 +108,17 @@ type Volume struct {
 }
 
 type BackupVolume struct {
-	Name             string                             `json:"name"`
-	Size             string                             `json:"size"`
-	Labels           map[string]string                  `json:"labels"`
-	Created          string                             `json:"created"`
-	LastBackupName   string                             `json:"lastBackupName"`
-	LastBackupAt     string                             `json:"lastBackupAt"`
-	DataStored       string                             `json:"dataStored"`
-	Messages         map[backupstore.MessageType]string `json:"messages"`
-	Backups          map[string]*Backup                 `json:"backups"`
-	BackingImageName string                             `json:"backingImageName"`
-	BackingImageURL  string                             `json:"backingImageURL"`
-
-	// Deprecated
-	BaseImage string `json:"baseImage"`
```

Comment:
I think it's safe to remove it.
1. This field is never been used since we remove the backing image feature in v0.5.0. The backing image info is tracked by the new field `BackingImageName` after re-introducing backing image feature in v1.1.0.
2. Even if somehow the backup meta info json string contains this removed field, this field will be ignored during marshaling.

Reply:
Will call it out in the comit comment.